srcpkgs/dracut/files/kernel-uefi-hook-postinst

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Kernel post-install hook for dracut EFI bundle.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+. "${ROOTDIR}/etc/default/dracut-uefi-hook"
+if [ -z "${CREATE_UEFI_BUNDLES}" ]; then
+	exit 0
+fi
+
+if [ ! -x bin/dracut ]; then
+	exit 0
+fi
+
+if [ -z "${KERNEL_CMDLINE}" ] && [ "x${SKIP_KERNEL_CMDLINE}" != x1 ]; then
+	echo "ERROR: You need to set the kernel command line in /etc/default/dracut-uefi-hook"
+	exit 1
+fi
+
+DRACUT_OPTIONS="${DRACUT_OPTIONS} --force --uefi ${UEFI_BUNDLE_DIR:=boot/efi/EFI/void}/linux-${VERSION}.efi ${VERSION}"
+mkdir -p "${UEFI_BUNDLE_DIR}"
+
+dracut -q --kernel-cmdline="${KERNEL_CMDLINE}" ${DRACUT_OPTIONS}
```

Comment:
```sh
dracut -q ${KERNEL_CMDLINE:+--kernel-cmdline="${KERNEL_CMDLINE}"} ${DRACUT_OPTIONS}
```
looks better?

Reply:
I think the issue is that, if you don't set `kernel_cmdline` in the config file or the `dracut` command line, the user will be prompted for a value, which will stall the hook. @ericonr, is this correct?

Reply:
It does!

Reply:
Gah I hadn't seen @ahesford's comment.

So, that one does look better, and if I add the `-q` argument to dracut it simply plows ahead and gets an empty cmdline. So it should be fine.